raiden_installer/installer_parts/funding.py

```diff
@@ -0,0 +1,173 @@
+import hashlib
+import requests
+import json
+from uuid import getnode
+from getpass import getuser
+from raiden_contracts.contract_manager import (
+    ContractManager,
+    contracts_precompiled_path,
+    get_contracts_deployment_info
+)
+from raiden_contracts.constants import (
+    CONTRACT_USER_DEPOSIT,
+    CONTRACT_CUSTOM_TOKEN
+)
+
+
+def goerli_funding(address: str) -> None:
+    '''
+    Fund address with Gö-ETH from faucet
+    '''
+    mac_address = getnode()
+    current_user = getuser()
+    client_hash = hashlib.sha256(
+        f'{mac_address}-{current_user}'.encode()
+    ).hexdigest()
+
+    try:
+        res = requests.post(
+            'https://faucet.workshop.raiden.network/',
+            json={'address': address, 'client_hash': client_hash}
+        )
+        print(res)
+    except requests.exceptions.RequestException as err:
+        print(err)
+
+
+class PfsAndMonitoringFunding:
```

Comment:
This would be a good question to discuss. I made it a class mostly to organize the code but also not to create tons of functions that return something that then needs to be passed somewhere else etc. But I’m not sure this is the best approach and would love to hear feedback on a better solution.